python/mxnet/image.py

```diff
@@ -71,7 +95,45 @@ def scale_down(src_size, size):
 
 
 def resize_short(src, size, interp=2):
-    """Resize shorter edge to size."""
+    """Resizes shorter edge to size.
+
+    Note: `resize_short` uses OpenCV (not the CV2 Python library).
+    MXNet must have been built with OpenCV for `resize_short` to work.
+
+    Resizes the original image by setting the shorter edge to size
+    and setting the longer edge accordingly.
+    Resizing function is called from OpenCV.
+
+    Parameters
+    ----------
+    src : NDArray
+        The original image.
+    size : int
+        The number to be set for the shorter edge.
```

Comment:
@zackchase I changed the word 'number' to 'length' in the doc, please see if it satisfies the requirement.